data/json/items/gun/40mm.json

```diff
@@ -16,12 +41,63 @@
     "range": 30,
     "dispersion": 180,
     "sight_dispersion": 120,
-    "aim_speed" : 6,
+    "aim_speed": 6,
     "recoil": 270,
     "durability": 8,
     "burst": 10,
     "reload": 400,
     "magazines": [ [ "40mm", [ "belt40mm" ] ] ],
     "flags": "MOUNTED_GUN"
+  },
+  {
+    "id": "rm802",
+    "type": "GUN",
+    "name": "RM802 grenade launcher",
+    "description": "The Rivtech RM802 pump-action 40mm grenade launcher was designed to be a step ahead of single-shot grenade launchers by allowing sustained heavy firepower.  However, its capacity is fairly limited so it must be used with careful planning.",
+    "weight": 3120,
+    "volume": 6,
+    "price": 2920000, "//": "Either Rivtech or milspec would make it Expensive.  Combined, you could buy a half-decent car for the price.",
+    "bashing": 8,
+    "to_hit": -1,
+    "material": [ "superalloy", "ceramic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "skill": "launcher",
+    "ammo": "40mm",
+    "dispersion": 150,
+    "sight_dispersion": 120,
+    "aim_speed": 3,
+    "recoil": 30,
+    "durability": 9,
+    "clip_size": 4,
+    "reload": 100,
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
+    "flags": [ "RELOAD_ONE", "WATERPROOF_GUN", "NEVER_JAMS" ]
+  },
+  {
+    "id": "triple_launcher_simple",
+    "type": "GUN",
+    "name": "triple-barrel 40mm launcher",
+    "description": "A homemade triple-barrel 40mm grenade launcher.  Capable of launching all three grenades at once, it is extremely powerful for a homemade weapon.",
+    "weight": 3759,
+    "volume": 10,
+    "price": 95000,
+    "bashing": 12,
+    "to_hit": -3,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "skill": "launcher",
+    "ammo": "40mm",
+    "dispersion": 300,
+    "sight_dispersion": 150,
+    "aim_speed": 0,
+    "recoil": 60,
+    "durability": 7,
+    "burst": 3,
+    "clip_size": 3,
+    "reload": 350,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]],
```

Comment:
I am not going to do it in this PR, but how could I separate the underbarrel hardpoint into bayonet and underbarrel hardpoints? It would stop the arguments about flintlock-grenade launcher combinations, and it would allow guns to have bayonets and grenade launchers simultaneously, which is possible irl.